tests/tracker-health-auth-test.php

```
@@ -0,0 +1,193 @@
+    }
+}
+
+if (!function_exists('get_option')) {
+    function get_option(string $option, $default = false)
+    {
+        return $GLOBALS['_tha_options'][$option] ?? $default;
+    }
+}
+
+if (!function_exists('register_rest_route')) {
+    function register_rest_route(string $namespace, string $route, array $args): void {}
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$namespace'. (undefined)

(UnusedFormalParameter)